src/utils/index.js

```diff
@@ -52,10 +55,13 @@ export const ValidationReasons = Object.freeze({
     EMAIL: "Must be a valid email.",
     HAVE_NUMBER: "Must contain at least a number.",
     ALREADY_EXISTS: (variable) => `${variable} already exists.`,
-    DATE_EXPIRED: "Date must not be in the past",
+    DATE_EXPIRED: "Date must not be in the past.",
     MUST_BE_AFTER: (variable) => `Date must be after ${variable}.`,
     FILE_TOO_BIG: (val) => `File size must be under ${val}.`,
     FILE_TYPE_ALLOWED: (vals) => `File type must be one of the following: ${vals.join(", ")}.`,
+    LOCATION_FORMAT: () => "The location format must be <city>, <country>. Beware of extra spaces.",
+    PUBLISH_END_DATE: () => `Publication end date should be after Publication date and last less 
```

Comment:
Dates don't "last". Offers do 

```suggestion
    PUBLISH_END_DATE: () => `Publication end date should be after Publish Date but not over ${OFFER_MAX_LIFETIME_MONTHS} month(s) after that.
```